### ISSUE TITLE
Album art broken

### ISSUE THREAD

**Issue by rechner** [AzuraCast/AzuraCast#2179]
**Using Docker installation method**
Yes
 v0.9.7.1, #03f7423 (2019-10-31 15:09)

**Host Operating System**
Debian 10

**Describe the bug**
After some time with everything working smoothly, importing songs from either the web interface or ssh results in a broken album art preview.

![](https://i.imgur.com/aUgV6O9.png)

Clicking on the preview results in this error:

![](https://i.imgur.com/Bu45JQB.png)

The web request for each image don't result in any error, just 200 OK and `Content-Length: 0`.

Digging into the file system, my `<station_data>/<station>/album_art/` is full of 0 byte .jpg files.

**To Reproduce**
1. Go to station files
2. Click and drag music with ID3 album art embedded
3. Check file after it's processed
4. Notice broken album art

**Expected behavior**
Album art should be shown for each song, or replaced by the default, "doesn't have album art" icon.

**Relevant Logs**
Web logs reveal nothing remarkable, other than info about buffering the file upload:
```
web_1       | 2019/11/28 03:36:10 [warn] 74#74: *873 a client request body is buffered to a temporary file /tmp/azuracast_nginx_client/6/5/0000000456, client: 10.0.0.21, server: localhost, request: "POST /api/station/2/files/upload HTTP/1.1", host: "radio.knot.space", referrer: "https://my.station/station/2/files"
...
web_1       | [2019-11-28T03:40:01.929192+00:00] AzuraCast.INFO: New message of type App\Message\AddNewMediaMessage added to queue. [] []
web_1       | [2019-11-28T03:40:01.929663+00:00] AzuraCast.INFO: New message of type App\Message\AddNewMediaMessage added to queue. [] []
web_1       | [2019-11-28T03:40:01.940345+00:00] AzuraCast.INFO: Handling message of type App\Message\AddNewMediaMessage. [] []
web_1       | [2019-11-28T03:40:08.524058+00:00] AzuraCast.INFO: Handling message of type App\Message\AddNewMediaMessage. [] []
```

**Screenshots**

**Device(s):**
 - Device: Desktop
 - OS: Debian 10
 - Browser: Brave
 - Version: 0.64.77

**Additional context**

I had album art working for a fleeting moment after the initial setup, but no amount of wiping out and recreating stations get me back to a working state.

**Comment by rmacari** [comment]
Same error here. 
Temp fix: Don´t use https, use only http in your url website, it looks like that is something related to the SSL certificate.

**Comment by rechner** [comment]
Hmmm, I do use a reverse proxy for this, I'll fiddle more do see if any of those settings change behaviour

**Comment by SlvrEagle23** [comment]
@rechner If you don't mind, could you join either our [Discord](https://azuracast.com/discord) or [Slack](https://azuracast.com/slack) communities so we can look into this further?

There's definitely something wrong with the album artwork parsing, and I'd like to investigate the issue more. This may be due in some way to our updates to PHP 7.4 (and underlying updates to the `getid3` library), though I can't reproduce the same issue locally when testing.

**Comment by hanamiyuna** [comment]
I got this issue today.

If I make the reverse proxy to `https://127.0.0.1:8443` for example, the static resources are failed to load. This will still happen after I set reverse proxy subdomain.domain.com to `http://127.0.0.1:8080`(just a example). And here may come to a strike problem where my links all became `https://127.0.0.1:8443/path` due to the settings.

However, the album art still can be found in the Music Files section.
